scripts/generate-dotnet.sh

```diff
@@ -47,8 +47,16 @@ if [ "$PLATFORM" = "ghec" ]; then
 elif [ "$PLATFORM" = "ghes" ]; then
 	NAMESPACE="dotnet-sdk-enterprise-server"
     CSPROJ_PACKAGE_FILE="stage/dotnet/$NAMESPACE/src/GitHub.Octokit.GHES.SDK.csproj"
+    BUILD_YAML="stage/dotnet/$NAMESPACE/.github/workflows/build.yml"
+
+    # Update the PackageId in the .csproj file to include the version
     sed -i "s|<PackageId>GitHub.Octokit.GHES.SDK</PackageId>|<PackageId>GitHub.Octokit.GHES.SDK.$VERSION</PackageId>|" "$CSPROJ_PACKAGE_FILE"
     echo "Updated <PackageId> to GitHub.Octokit.GHES.SDK.$VERSION in $CSPROJ_PACKAGE_FILE"
+
+    # Update the branches node in the publish.yml file to include the current version only
+    # This will set the branches node to an array containing only the new branch name, effectively replacing anything under the node.
+    sed -i'' "s/- main/- $VERSION/" "$BUILD_YAML"
```

Comment:
Have you validated this in Actions/on a Linux machine yet? I know there's occasionally differences between your Mac environment and on Actions Ubuntu machines for these types of commands.

Reply:
I did run them on a linux env - the actions vm specifically.  It uses identical params as the one above it.